thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyMergeConfig.java

```diff
@@ -39,7 +39,10 @@
 public class AnomalyMergeConfig {
   private AnomalyMergeStrategy mergeStrategy = AnomalyMergeStrategy.FUNCTION;
   private long sequentialAllowedGap = 30_000; // 30 seconds
-  private long maxMergeDurationLength = 12 * 60 * 60 * 1000; // 12 hours
+
+  // Look back for 2 days to accommodate merging of anomalies in daily metrics
+  private long maxMergeDurationLength = 48 * 60 * 60 * 1000; // 48 hours
```

Comment:
Well there is no perfect number, but if we set it to something big, say 7 days, then we might end up merging anomalies and have an anomaly which spans for 7 days. This could end up swallowing critical alerts which we may want to avoid.